docs/data_pipeline_procedure.md

```diff
@@ -0,0 +1,92 @@
+# Data Pipeline Procedure
+
+## Download Source Data
+   * Workspace Downloader
+     * Example usage
+        ```commandline
+        python src/loaders/workspace_downloader/workspace_downloader.py \
+          --env $env \
+          --workspace_id $workspace_id \
+          --kbase_collection $kbase_collection \
+          --source_ver $source_verion \
+          --token_filepath $token_filepath \
+          --retrieve_sample 
+        ```
+     * Generated files/metafiles
+       * downloaded files
+         ```text
+         sourcedata/WS/[env]/[workspace_id]/[UPA]/
+         ```
+       * Metadata files
+         ```text
+         sourcedata/WS/[env]/[workspace_id]/[UPA]/[UPA].meta
+         ```
+       * Softlinks for collections
+         ```text
+         collectionssource/[env]/[kbase_collection]/[source_ver]/[UPA]/
+         ```
+
+   * NCBI Downloader
+     * Example usage
+       ```commandline
+       python src/loaders/ncbi_downloader/gtdb.py \
+         --download_file_ext $download_file_ext \
+         --release_ver $release_ver
+       ```
+     * Generated files/metafiles
+       * downloaded files
+         ```text
+         sourcedata/NCBI/NONE/[ncbi_genome_id]/
+         ```
+       * Metadata files
+         ```text
+         None
+         ```
+       * Softlinks for collections
+         ```text
+         collectionssource/NONE/GTDB/[GTDB_release_ver]/[ncbi_genome_id]/
+         ```
+## Schedule Taskfarmer Jobs
+   * Example usage
+     ```commandline
+     python src/loaders/jobs/taskfarmer/task_generator.py \
+       --tool $tool \
+       --env $env \
+       --kbase_collection $kbase_collection \
+       --source_ver $source_ver \
+       --load_ver $load_ver \
+       --source_file_ext $source_file_ext \
+       --submit_job
+     ```
+   * Generated files/metafiles
+     * Taskfarmer job files
+       ```text
+       task_farmer_jobs/[job_dir]/
```

Comment:
Can you explain the value of `job_dir` and what's contained in the dir?

Reply:
What is the actual value of `job_dir` though? Is it a job id? If so where does it come from?

Reply:
added.

Reply:
:+1: